### PR TITLE
add doc for bandit justification

### PR DESCRIPTION
**Pull request by KPostOffice**
Fixes: #182

### CODE REVIEW

File: _j/si_bandit.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+This is not an issue
+
+## Pipeline units
+
+ * [SecurityIndicators](https://thoth-station.ninja/docs/developers/adviser/thoth.adviser.steps.html#module-thoth.adviser.steps.security_indicators)
+
+## Recommendation types
+
+ * security
+
```
Comment (fridex) [inline _j/si_bandit.md:34]:
```suggestion
 * stable
```

https://github.com/thoth-station/adviser/blob/5bc484d83e978ab82e2745ece2ae8a7ba5332034/thoth/adviser/steps/security_indicators.py#L84-L88

**Review by fridex — COMMENTED** [review]
One minor comment, otherwise LGTM!! :+1:

File: _j/si_bandit.md (hunk 1 above)
Comment (KPostOffice) [reply _j/si_bandit.md:34]: https://github.com/thoth-station/adviser/blob/5bc484d83e978ab82e2745ece2ae8a7ba5332034/thoth/adviser/steps/security_indicators.py#L140

Comment (fridex) [reply _j/si_bandit.md:34]: Ah, ok. I see your point :+1:

**Review by fridex — APPROVED** [review]

**Pull request merged**